VisualPinball.Unity/Documentation~/creators-guide/tutorials/realistic-backglass/2-create-mesh.md

```diff
@@ -0,0 +1,41 @@
+---
+uid: tutorial_backglass_2
+title: Realistic Looking Backglass - Create Mesh
+description: How to create a Backglass mesh in Blender
+---
+
+# Create Mesh
+
+We're now going to import edit a cube and unwrap it to fit the backglass artwork.
+
+## Step 1: Edit the Cube
+
+Open Blender, choose File/New/General and select the cube that is native to this scene while in Object Mode.  Under the Scene setting in the lower right change the units to "Imperial" and set the length to "Inches".  At the upper right of the window pull out the menu there and type in the exact dimensions of your backglass.
```

Comment:
When referring to screen to screen elements, I write them in italic and separate them with "->", which renders as a nice arrow.

```suggestion
Open Blender, choose *File -> New -> General* and select the cube that is native to this scene while in Object Mode.  Under the Scene setting in the lower right change the units to *Imperial* and set the length to *Inches*.  At the upper right of the window pull out the menu there and type in the exact dimensions of your backglass.
```

Also, isn't Blender's cube called the "default cube"? It's funny actually, this is the first Blender tutorial I come across where the first action isn't "delete the default cube". :)

Reply:
I know so little about Blender its funny that I have the audacity to write a tutorial!  I'm a make it in F360 and print it, lay it out in v-carve and CNC it, design it in Lightburn and laser cut it guy.  Don't make it pretty, just make it!